flask/app.py

```diff
@@ -0,0 +1,107 @@
+"""
+author: Robert Beatty
+date: March 11, 2019
+purpose: simple REST API to retrieve summary information in DVRPC's crash data tool
+"""
+
+
+from flask import Flask, request
+from flask_cors import CORS
+from lib import credentials
+import psycopg2 as psql
+from psycopg2 import sql
+import json
+
+
+
+app = Flask(__name__)
+CORS(app)
+
+def connectToPsql():
+    return psql.connect(credentials.PSQL_CREDS)
+
+"""
+@TODO:  create documentation page using flask's render_template function to deliver an HTML file
+        to give details about how to use the API
+"""
+@app.route('/api/crash-data/v1/documentation')
+def docs():
+    return '<html><p>this will be the docs page</p></html>'
+
+
+@app.route('/api/crash-data/v1/popupInfo', methods=["GET"])
+def get_popup_info():
+    ## grab crash id
+    id = request.args.get('id')
+
+
+    ## connect to db
+    con = connectToPsql()
+    cur = con.cursor()
+    qry = """
+    SELECT 
```

Comment:
it is PERSON_COUNT, but I will change the column name in the table to reflect that because my quick naming scheme wasn't exactly intuitive.

Added collision type. 👍